docs/automate.md

```diff
@@ -0,0 +1,26 @@
+# Automate Configuring Self-Hosted Runner Pools
+
+## Latest Release as Service on Nix VMs
+
+[Run or copy this script for your use](../scripts/latest-svc.sh) to automate configuring a runner as a service on Linux or Mac
+
+### Step 1: export PAT
+
+Create a GitHub PAT and export it before running the script
```

Comment:
We might want to clarify the scopes required for the PAT so that users don't have to look at the API docs to know. I also don't think we need to specify a "GitHub PAT".

```suggestion
Create a PAT with `repo` scopes for repo runners and `admin:org` scopes for org runners. Export this PAT as an environment variable before running the script.
```

Reply:
Cool.  Is there somewhere in docs that clarifies the minimum scopes?  Ideally, I would just link to docs.  But agreed, good addition.